wp-content/plugins/hurumap/src/ChartDefinition/HurumapChart.js

```diff
@@ -229,6 +229,32 @@ function HurumapChartDefinition({ chart, data, sectionOptions, onChange }) {
                   <Grid item>Horizontal</Grid>
                 </Grid>
               )}
+              {visual.type && visual.type.includes('line') && (
+                <Grid
+                  item
+                  container
+                  component="label"
+                  alignItems="center"
+                  spacing={1}
+                >
+                  <Grid item>Interpolation: Linear</Grid>
+                  <Grid item>
+                    <Switch
```

Comment:
Lets switch to a dropdown with all the options: https://formidable.com/open-source/victory/docs/victory-line/#interpolation

Reply:
Cool